src/proxy-ui-api/frontend/tests/e2e/specs/ss-clients-localgroups.js

```diff
@@ -0,0 +1,60 @@
+
+module.exports = {
+  tags: ['ss', 'clients', 'localgroups'],
+  'Security server client local groups filtering': browser => {
+    const frontPage = browser.page.ssFrontPage();
+    const mainPage = browser.page.ssMainPage();
+    const clientsTab = mainPage.section.clientsTab;
+    const clientInfo = mainPage.section.clientInfo;
+    const clientDetails = clientInfo.section.details;
+    const clientInternalServers = clientInfo.section.internalServers;
+    const clientLocalGroups = clientInfo.section.localGroups;
+    const certificatePopup = mainPage.section.certificatePopup;
+
+    // Open SUT and check that page is loaded
+    frontPage.navigate();
+    browser.waitForElementVisible('//*[@id="app"]');
+
+    // Enter valid credentials
+    frontPage
+      .clearUsername()
+      .clearPassword()
+      .enterUsername(browser.globals.login_usr)
+      .enterPassword(browser.globals.login_pwd)
+      .signin();
+
+    // Open TestGov Internal Servers
+    mainPage.openClientsTab();
+    browser.waitForElementVisible(clientsTab);
+    clientsTab.openTestService();
+    browser.waitForElementVisible(clientInfo);
+    clientInfo.openLocalGroupsTab();
+    browser.waitForElementVisible(clientLocalGroups);
+    // Verify default sorting
+    browser
+      .useXpath()
+      .waitForElementVisible('(//table[contains(@class, "details-certificates")]/tr)[2]//span[contains(text(),"1122")]')
```

Comment:
Similar xpath appears on many places, would it make sense to implement helper function, something like `detailsTableContains(int row, String textValue)` ?